cmd/tail/tail.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"teleport-exec/filestream"
+	"time"
+)
+
+//-------------------------------------------------------------------------------------------------
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatalln("Need an argument!")
+	}
+	file_name := os.Args[1]
+	log.Println("Tailing file:", file_name)
+
+	// Timeout tailing after a while
+	timeout_ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	// Stop the stream when cancelled via Ctrl+C
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
```

Comment:
Did you consider using`signal.NotifyContext` and passing in the timeoutCtx as the parent?

Reply:
TIL about `signal.NotifyContext` 👍🏻